### PR TITLE
Replace json assets by js files

### PR DESCRIPTION
**Pull request by etpinard**
@mdtusz @bpostlethwaite 

Webpack does not bundle up json files by default. It is possible using user-defined [loaders](https://webpack.github.io/docs/using-loaders.html) e.g. this [one](https://github.com/webpack/json-loader) for json files. But this is an extra hurdle for webpack users that we can *fix* on our end.

This PR takes one step towards supporting Webpack bundling without any special configs.

Note that I used this [script](https://gist.github.com/etpinard/bac3656466458607ddd0) to check for non-js files that are bundled into plotly.js.

### CODE REVIEW

**Comment by etpinard** [comment]
TODO:

- [ ] make a PR to https://github.com/bpostlethwaite/colormap
- [ ] find a way to inject the package version into the plotly.js index without requiring the package.json

File: src/constants/gl_markers.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+module.exports = {
+    circle: '●',
+    'circle-open': '○',
+    square: '■',
+    'square-open': '□',
+    diamond: '◆',
+    'diamond-open': '◇',
+    cross: '+',
+    x: '❌'
+};
```
Comment (mdtusz) [inline src/constants/gl_markers.js]: :cow2:, but I guess it's unavoidable here.

File: tasks/header.js [hunk 2]
```diff
@@ -43,6 +43,11 @@ glob(path.join(constants.pathToSrc, '**/*.js'), function(err, files) {
 
             var header = comments[0];
 
+            // error out if no header is found
+            if(!header || header.loc.start.line > 1) {
+                throw new Error(file + ' : has no header information.');
+            }
+
```
Comment (mdtusz) [inline tasks/header.js]: :smile_cat:

**Comment by mdtusz** [comment]
:dancer:

**Comment by mdtusz** [comment]
Did this actually get merged? It shows up as such, but is still an open PR

File: src/constants/gl_markers.js (hunk 1 above)
Comment (etpinard) [inline src/constants/gl_markers.js]: yes unfortunately: https://github.com/gl-vis/gl-scatter3d#example

**Pull request merged**